src/vm/loaderallocator.hpp

```diff
@@ -273,6 +273,10 @@ class LoaderAllocator
     CrstExplicitInit m_InteropDataCrst;
     EEMarshalingData* m_pMarshalingData;
 
+    // Used for synchronizing access to m_stubMethodInfoCache.
+    CrstExplicitInit m_stubMethodInfoCacheCrst;
+    HashMap m_stubMethodInfoCache;
```

Comment:
Do not use HashMap directly for holding pointers. Per the comments on the PtrHashMap implementation, you should never put use a pointer as a key in a HashMap, you should always use a PtrHashMap. Of course, you can't safely store a LOADERHANDLE in a PtrHashMap, as it requires the low bit to be null on values. Instead I recommend you use a MapSHash.